README.md

```diff
@@ -465,6 +469,23 @@ You can use `SwiftLocation.preciseAccuracy` to get the current precise authoriza
 This option is available since iOS14+. Older versions always uses `fullAccuracy` and the parameter is ignored.****
 
 
+### Async/Await Example
+
+To take advantage of Swift's async/await concurrency pattern, use the `async(queue:)` method of the `RequestProtocol` in place of `then(queue:)`.  Note that the _queue_ argument is optional and defaults to `DispatchQueue.main`.
+
+```
```

Comment:
Hi @levous could you help me by checking if this still works? I am not able to run gpsLocationWith as async. Thank you very much in advance, Joao